### PR TITLE
Remove ksonnet references from miscellaneous GKE docs

### PR DESCRIPTION
**Pull request by richardsliu**
Part of https://github.com/kubeflow/website/issues/929

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/938)

### CODE REVIEW

File: content/docs/gke/custom-domain.md [hunk 1]
```diff
@@ -17,19 +17,31 @@ so, follow the guide to
...
 1. Set the domain for your ingress to be the fully qualified domain name:
 
     ```
-    ks param set iap-ingress hostname ${FQDN}
-    ks apply default -c iap-ingress
+    cd ${KFAPP}/kustomize
+    gvim iap-ingress.yaml    # Or basic-auth-ingress.yaml
+    ```
+
+   Find and replace the value for the hostname:
```
Comment (sarahmaddox) [inline content/docs/gke/custom-domain.md]: Apply indentation to fix list formatting:
https://deploy-preview-938--competent-brattain-de2d6d.netlify.com/docs/gke/custom-domain/

File: content/docs/gke/private-clusters.md [hunk 2]
```diff
@@ -331,8 +330,20 @@ export PROJECT_NUMBER=$(gcloud projects describe kubeflow-dev --format='value(pr
 1. Update iap-ingress component parameters:
 
     ```
-    cd ${KFAPP}/ks_app
-    ks param set iap-ingress privateGKECluster true
+    cd ${KFAPP}/kustomize
+    gvim basic-auth-ingress.yaml  # Or iap-ingress.yaml if you are using IAP
+    ```
+
+   Find and set the `privateGKECluster` parameter to true:
```
Comment (sarahmaddox) [inline content/docs/gke/private-clusters.md]: Apply indentation to fix list formatting: 
https://deploy-preview-938--competent-brattain-de2d6d.netlify.com/docs/gke/private-clusters/

**Comment by richardsliu** [comment]
@sarahmaddox Fixed:
https://deploy-preview-938--competent-brattain-de2d6d.netlify.com/docs/gke/custom-domain/
https://deploy-preview-938--competent-brattain-de2d6d.netlify.com/docs/gke/private-clusters/

**Pull request merged**